src/__experimental__/components/form/form.component.js

```diff
@@ -218,21 +220,19 @@ class FormWithoutValidations extends React.Component {
         this.addInputDataToState(inputName, value);
       }
     });
-    this.submitControlledForm();
+    Service.configure({ csrfToken: this.csrfToken });
```

Comment:
shouldn't that token be set globally in another part of the application?

Reply:
It's set when by the `generateCSRFToken` function when the form renders, not sure how else to approach setting the token @chrisbarber86 any thoughts?